packages/musl_libbacktrace.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from Arch Linux libbacktrace-git PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=libbacktrace-git
+
+require 'package'
+
+class Musl_libbacktrace < Package
+  description 'Library to produce symbolic backtraces'
+  homepage 'https://github.com/ianlancetaylor/libbacktrace'
+  version 'd0f5e95a87a4d3e0a1ed6c069b5dae7cbab3ed2a'
+  compatibility 'all'
+  source_url 'https://github.com/ianlancetaylor/libbacktrace.git'
+  git_hashtag version
+
```

Comment:
No binaries?

Reply:
this is entirely for testing on i686, and I'm not going to generate binaries for it...